op-e2e/system_test.go

```diff
@@ -31,6 +32,8 @@ import (
 	"github.com/ethereum-optimism/optimism/op-node/withdrawals"
 )
 
+var enableParallelTesting bool = true
```

Comment:
Why not use the `-p 1` go testing flag to limit parallelism?

Reply:
`-p 1` does not stop tests from being run in parallel, it simply multiplexes them over a single process.